common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -98,7 +98,9 @@ def truncate_pr_message(msg)
           msg = (msg[0..trunc_length] + tr_msg)
         end
         # if we used a custom encoding for calculating length, then we need to force back to UTF-8
-        msg.force_encoding(Encoding::UTF_8) unless pr_message_encoding.nil?
+        # Using msg.force_encoding(Encoding::UTF_8) doesn't always work.
+        # https://stackoverflow.com/a/10466273
```

Comment:
We already have the test and the git diff that explain this, I don't think the comment is necessary

```suggestion
```

Reply:
Hopefully, someone doesn't wonder why it's the case